ventor_base/migrations/14.0.1.3.6/end-set_warehouses_on_inventory.py

```diff
@@ -0,0 +1,15 @@
+# Odoo:
+from odoo import SUPERUSER_ID, _, api
+
+
+def migrate(cr, version):
+    """Update warehouse_id for all stock inventory"""
+
+    env = api.Environment(cr, SUPERUSER_ID, {})
+    all_stock_inventory = env['stock.inventory'].search([
+            ('state', '=', 'draft'),
+            ])
```

Comment:
we need to also update non-draft inventories to hide them

Reply:
Done